TestVectors/dafny/TestVectors/src/ParseEsdkJsonManifest.dfy

```diff
@@ -108,9 +113,13 @@ module {:options "-functionSyntax:4"} ParseEsdkJsonManifest {
     var obj := json.obj;
 
     match version
-    case 3 =>
```

Comment:
Seems like you removed `case 3` -- do we still need this?
Ditto on some V3 stuff below

Reply:
v3 is not needed